### PR TITLE
update taiwaneseMandarin.xml to v8.1.9.2

### PR DESCRIPTION
**Pull request by killgavin**
update taiwaneseMandarin.xml to v8.1.9.2

### CODE REVIEW

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml [hunk 1]
```diff
@@ -314,8 +314,9 @@
                     <Item id="10004" name="在新執行個體開啟"/>
 
                     <Item id="46001" name="語法樣式配置器..."/>
-                    <Item id="46250" name="定義你的語法樣式..."/>
+                    <Item id="46250" name="定義您的語法樣式..."/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:317]: 「您」這個字在 Notepad++ 的字典中並不存在，請您把「您」還原成「你」。

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml [hunk 2]
```diff
@@ -1047,11 +1050,9 @@
                     <Item id="6153" name="允許多重實體"/>
                     <Item id="6154" name="禁止多重實體（預設）"/>
                     <Item id="6155" name="（重新啟動 Notepad++ 以使變更生效。）"/>
-                    <Item id="6171" name="插入日期時間自訂"/>
-                    <Item id="6175" name="顛倒日期時間內定順序（簡短與完整格式）"/>
+                    <Item id="6171" name="自訂插入日期時間"/>
+                    <Item id="6175" name="反轉默認日期時間順序（簡短與完整格式）"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:1054]: 「內定」is already used in all Taiwanese Mandarin translation, so please use 「內定」instead.

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml [hunk 3]
```diff
@@ -1157,18 +1158,18 @@
                 <Item id="1720" name="「.」包含換行字元"/>
             </FindInFinder>
             <DoSaveOrNot title="儲存檔案"> 
-                <Item id="1761" name="你要儲存「$STR_REPLACE$」嗎？"/>
-                <Item id="6" name="好吧(&amp;Y)"/>
-                <Item id="7" name="不要啦(&amp;N)"/>
+                <Item id="1761" name="您要儲存「$STR_REPLACE$」嗎？"/>
+                <Item id="6" name="是(&amp;Y)"/>
+                <Item id="7" name="否(&amp;N)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:1163]: I kind of like 好吧 &不要啦, so please keep them.

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml [hunk 4]
```diff
@@ -1392,8 +1393,8 @@
             <finder-clear-all value="全部清除"/>
             <finder-open-all value="全部開啟"/>
             <finder-wrap-long-lines value="自動換行"/>
-            <common-ok value="OK啦"/>
-            <common-cancel value="取消啦"/>
+            <common-ok value="確定"/>
+            <common-cancel value="取消"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:1397]: Please keep them.

**Review by donho — CHANGES_REQUESTED** [review]
Please follow the review comments for all the requests.

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml (hunk 2 above)
Comment (donho) [reply PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:1054]: BTW 「預設」 is OK; but not 「默認」

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml [hunk 5]
```diff
@@ -430,7 +431,7 @@
             <IncrementalFind title="">
                 <Item id="1681" name="搜尋："/>
                 <Item id="1685" name="區分大小寫"/>
-                <Item id="1690" name="醒目提示所有"/>
+                <Item id="1690" name="所有醒目提示"/>
```
Comment (mingtsay) [inline PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:434]: 醒目提示所有符合之項目，原本的順序比較符合功能目的，是否考慮還原此字串修改？

**Review by mingtsay — CHANGES_REQUESTED** [review]

**Comment by donho** [comment]
Close due to the absence of reply.

**Pull request closed without merging**